Node/Generator/test/bot_js.test.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
```

Comment:
'bot_js.test.js' and 'bot_ts.test.js' are a refactor and expansion of the old 'app.js' test file.
Each generate a bot (js and ts) and validate the existence of specific files and methods in the files before cleaning up.